build.gradle.kts

```diff
@@ -6,3 +6,12 @@ plugins {
     `binary-compatibility-validator`
     `aps-plugin`
 }
+
+subprojects {
+    configurations.all {
+        resolutionStrategy.dependencySubstitution {
+            substitute(module("org.jetbrains.trove4j:trove4j:20160824"))
+                .using(module("org.jetbrains.intellij.deps:trove4j:1.0.20200330"))
```

Comment:
This is the new module used by AGP in 7.0.0, since it's going to be a while we can upgrade to it I simply substitute it in dependency resolution and can confirm it works fine.